doc/installation.md

```diff
@@ -106,41 +106,10 @@ binaries are available for Linux and MacOS only.
 You can download the binaries in the [releases
 page](https://github.com/jolicode/castor/releases):
 
-=== "Linux AMD64 (x86-64)"
-
-    ```bash
-    curl "https://github.com/jolicode/castor/releases/latest/download/castor.linux-amd64" -Lfso $HOME/.local/bin/castor && \
-        chmod u+x $HOME/.local/bin/castor && \
-        castor --version || \
-        (echo "Could not install castor. Is the target directory writeable?" && (exit 1))
-    ```
-
-=== "Linux ARM64"
-
-    ```bash
-    curl "https://github.com/jolicode/castor/releases/latest/download/castor.linux-arm64" -Lfso $HOME/.local/bin/castor && \
-        chmod u+x $HOME/.local/bin/castor && \
-        castor --version || \
-        (echo "Could not install castor. Is the target directory writeable?" && (exit 1))
-    ```
-
-=== "macOS with Apple Silicon (M1, M2, M3)"
-
-    ```bash
-    curl "https://github.com/jolicode/castor/releases/latest/download/castor.darwin-arm64" -Lfso /usr/local/bin/castor && \
-        chmod u+x /usr/local/bin/castor && \
-        castor --version || \
-        (echo "Could not install castor. Is the target directory writeable?" && (exit 1))
-    ```
-
-=== "macOS with Intel"
 
-    ```bash
-    curl "https://github.com/jolicode/castor/releases/latest/download/castor.darwin-amd64" -Lfso /usr/local/bin/castor && \
-        chmod u+x /usr/local/bin/castor && \
-        castor --version || \
-        (echo "Could not install castor. Is the target directory writeable?" && (exit 1))
-    ```
+```bash
+curl "https://castor.jolicode.com/install" | bash -s -- --static
+```
```

Comment:
It misses a sentence before the exemple, because the previous paragraph is about downloading binaries from github IMO

Reply:
I switch both line, and edited a bit the sentence.
I should be better